fitbenchmarking/templates/table_template.html

```diff
@@ -68,6 +68,24 @@ <h3>Errors</h3>
                         </tr>
                     {% endfor %}
                 </table>
+		{% if has_pp %}
```

Comment:
Can you change the indenting on this section?

Reply:
fixed